### PR TITLE
add  Book and Label classes && can be archived and add item methods

### PR DESCRIPTION
**Pull request by tiagomarin**
## this is a sub-branch of book-and-label.  
**I'm merging into that one, not Development.**

## Project Requirements Tackled!

- [ ]  Create a Book class in a separate .rb file.

   - All Book class properties visible in the diagram should be defined and set up in the constructor method.
      - can_be_archived?() in the Book class
          - should override the method from the parent class
          - should return true if parent's method returns true OR if cover_state equals to "bad"
          - otherwise, it should return false

- [ ] Create a Label class with an association to the Item class (in a separate .rb file).
  - All Label class properties visible in the diagram should be defined and set up in the constructor method.
  - Implement methods:
      - add_item method in the Label class
          - should take an instance of the Item class as an input
          - should add the input item to the collection of items
          - should add self as a property of the item object (by using the correct setter from the item object)

### CODE REVIEW

File: classes/label.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'securerandom'
+
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(title, id = SecureRandom.random_number(1000))
+    @id = id
+    @title = title
+    @color = color
```
Comment (CindyDorantes) [inline classes/label.rb:10]: - [ ] Kindly, add the color variable to the parameters that must be provided to create a new instance of Label. You have defined the instance variable, just add it to the initialize parenthesis 😉

**Review by CindyDorantes — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

# Hi @tiagomarin, 👋🏻👋🏻 

Great job so far!🔥🔥🔥
There are some little details that you still need to work on to go to the next step but you are almost there! so kindly fix the error so we can proceed with your project.💯💯✨✨✅✅✅



## Highlights ✨
- Code work as expected ✅✅✅
- Great commits 🔥🔥

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: classes/label.rb (hunk 1 above)
Comment (tiagomarin) [reply classes/label.rb:10]: OMG, I didn't see that, I could swear it was there! Thanks!

Comment (tiagomarin) [reply classes/label.rb]: Changes appplied!

**Review by CindyDorantes — APPROVED** [review]
## Hi @tiagomarin , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
# Congratulations! 🎉 

## To highlight  🎯  
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 


### Cheers and Happy coding!👏👏👏

**Pull request merged**